parallel-install/pkg/deployment/deployment_test.go

```diff
@@ -16,44 +17,46 @@ import (
 	"time"
 )
 
+func callbackUpdate(update ProcessUpdate) {
```

Comment:
What do you think about providing a function like this on the `deployment` pkg as a simple default that clients can use and learn from.
Someting like:
```go
    func DefaultUpdater() func(update ProcessUpdate) {
        return func(update ProcessUpdate) {
                 // Do simple logging of updates.
        }
    }
```

It would be used as follows by a client:
```go
installer, err := deployment.NewDeployment(installationCfg, overrides, cmd.K8s.Static(), deployment.DefaultUpdater())
```

Reply:
Good idea. I will add such a function